consensus/istanbul/backend/engine.go

```diff
@@ -323,8 +323,8 @@ func (sb *backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 	// if there is a vote to attach, attach it to the header
 	header.Vote = sb.governance.GetEncodedVote(sb.address, number)
 
-	// add validators in snapshot to extraData's validators section
-	extra, err := prepareExtra(header, snap.committee(header.ParentHash, sb.currentView.Load().(*istanbul.View)))
+	// add validators (council list) in snapshot to extraData's validators section
+	extra, err := prepareExtra(header, snap.validators())
```

Comment:
This would be the core change of this PR, "like an original code, I reverted it to save all validator list in the field." Did I get it right?

Reply:
Yes. This is core change of this PR.